src/pharmpy/modeling/basic_models.py

```diff
@@ -134,10 +148,92 @@ def create_basic_pk_model(
         if model.modelfit_results is not None
         else None,
     )
-    if modeltype == 'oral':
+
+    if (
+        modeltype == 'pk_oral'
+        or modeltype == 'pk_ivoral'
+        or (modeltype == 'basic_pk' and administration == 'oral')
+        or (modeltype == 'basic_pk' and administration == 'ivoral')
+    ):
         model = set_first_order_absorption(model)
         model = set_initial_estimates(model, {'POP_MAT': mat_init})
         model = add_iiv(model, list_of_parameters='MAT', expression='exp', initial_estimate=0.1)
+    if modeltype == 'pk_ivoral' or (modeltype == 'basic_pk' and administration == 'ivoral'):
+        # FIXME : Dependent on CMT column having 1 and 2 as values otherwise
+        # compartment structure don't match
+        model = add_bioavailability(model)
+        model = set_initial_estimates(model, {'POP_BIO': 0.5})
+
+        # Change BIO statement to BIO = LOG(POP_BIO / (1 - POP_BIO))
```

Comment:
Yes, a BIO and FBIO statement is added when using `add_bioavailability()`

Reply:
The transformations for bioavailability statements will be integrated into the `add_bioavailability()` function instead, using a new `logit_transformation` argument to avoid having the transformations here. These changes will be applied to the next commit.